Tests/Unit/Rng/SeedableRngUnitTests.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+@testable import MobileCoin
+import XCTest
+
+class SeedableRngUnitTests: XCTestCase {
+
+    func testSeedableFromSeedBytes() throws {
+        let seed1 = Data32(repeating: 5)
+        let rng1 = MobileCoinChaCha20Rng(seed32: seed1)
+
+        let seed2 = Data32(repeating: 10)
+        let rng2 = MobileCoinChaCha20Rng(seed32: seed2)
+        let rng3 = MobileCoinChaCha20Rng(seed32: seed2)
+
+        XCTAssertNotNil(rng1)
+        XCTAssertNotNil(rng2)
+        XCTAssertNotNil(rng3)
+
+        let val1 = rng1.next()
+        let val2 = rng2.next()
+        let val3 = rng3.next()
+
+        print("********** SEEDABLE RNG UNIT TESTS")
```

Comment:
Seems a little debug-ey, if this is meant to stay, lets make it more readable.

Reply:
ah, yes.. this was definitely debuggy... cleaning up